saleor/graphql/views.py

```diff
@@ -341,7 +341,8 @@ def format_error(cls, error):
         exc = error
         while isinstance(exc, GraphQLError) and hasattr(exc, "original_error"):
             exc = exc.original_error
-
+        if isinstance(exc, (AssertionError, ValueError)):
```

Comment:
Why we're catching `ValueError` as well? I'm afraid this might catch too many errors, e.g. those that are real bugs in our code that we should fix.

Reply:
Because of second bug [SALEOR-2814](https://app.clickup.com/t/2549495/SALEOR-2814)

Reply:
@fowczarek What do you think?